core/src/main/scala/org/bitcoins/core/wallet/builder/RawTxFinalizer.scala

```diff
@@ -344,3 +358,33 @@ object SubtractFeeFromOutputsFinalizer {
     (newOutputs ++ unchangedOutputs).sortBy(_._2).map(_._1).toVector
   }
 }
+
+case object ShuffleFinalizer extends RawTxFinalizer {
+
+  override def buildTx(txBuilderResult: RawTxBuilderResult)(implicit
+      ec: ExecutionContext): Future[Transaction] = {
+    ShuffleInputsFinalizer
+      .andThen(ShuffleOutputsFinalizer)
+      .buildTx(txBuilderResult)
+  }
+}
+
+case object ShuffleInputsFinalizer extends RawTxFinalizer {
```

Comment:
Can you add a scaladoc for what he purpose of this is? Same thing with `ShuffleOutputsFinalizer`